sympy/combinatorics/perm_groups.py

```diff
@@ -4861,6 +4861,24 @@ def polycyclic_group(self):
 
         return PolycyclicGroup(pc_sequence, pc_series, relative_order, collector=None)
 
+    def exponent(self):
+        """
+            Returns the exponent of a group
+
+            The exponent e of a group G is the lcm of the orders of its elements,
```

Comment:
There is an existing method in sympy to compute lcm.
```
exp = 1
for g in self.generators:
    exp = lcm(exp, g.order())
```

Reply:
Okay I will change it.

Reply:
Done

Reply:
Can you dedent the docstrings?

Reply:
I am not sure do we need to do it for docstring too?

Reply:
It’s only for the code convention